### PR TITLE
Modify S6469(docker): Migrate to LayC

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6469/docker/rule.adoc [hunk 1]
```diff
@@ -2,44 +2,73 @@
...
 
-[source,docker]
+[source,docker,diff-id=1,diff-type=compliant]
 ----
 RUN --mount=type=secret,id=build_secret,uid=1000 ./installer.sh
 ----
 
+=== How does this work?
+
+Always set the `others` bit to `0`.
```
Comment (sebastien-andrivet-sonarsource) [inline rules/S6469/docker/rule.adoc:55]: From the explanations, it is not clear if the default permissions (i.e. those when you do not specify `mode`, 'uid', ...) are safe or not.

File: rules/S6469/docker/rule.adoc [hunk 2]
```diff
@@ -2,44 +2,73 @@
...
 RUN --mount=type=secret,id=build_secret,mode=0777 ./installer.sh # Noncompliant
 ----
 
-=== Compliant solution
+==== Compliant solution
 
-[source,docker]
+[source,docker,diff-id=1,diff-type=compliant]
 ----
 RUN --mount=type=secret,id=build_secret,uid=1000 ./installer.sh
```
Comment (sebastien-andrivet-sonarsource) [inline rules/S6469/docker/rule.adoc:50]: For me, the compliant solution does not match the noncompliant one. We talk about permissions (i.e. `mode`) and the solution uses `uid`. It gives the impression that `uid` is also a permission with an octal value of `1000`.

**Review by sebastien-andrivet-sonarsource — CHANGES_REQUESTED** [review]
I find the explanation not very clear and somewhat misleading (it was also the case of the original rule). In particular, the example.

File: rules/S6469/docker/rule.adoc (hunk 2 above)
Comment (loris-s-sonarsource) [reply rules/S6469/docker/rule.adoc:50]: True, I didn't notice that

**Review by sebastien-andrivet-sonarsource — APPROVED** [review]
LGTM except for a link.

**Pull request merged**